templates/taxi/manufacturer_delete_confirm.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure that u want to delete the car. Others things which are somehow connected with this car will be
+    deleted.</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+    <a href="#" class="btn btn-secondary" onclick="history.back()">Cancel</a>
+  </form>
+{% endblock %}
```

Comment:
add new empty line

Reply:
The comment is not resolved, an empty line is absent